### DISCUSSION TITLE
should be className

### DISCUSSION THREAD

**Discussion by greenhand-xj** [chakra-ui/panda#765] [Show and tell]
https://panda-css.com/docs/concepts/patterns#divider

**Comment by anubra266** [comment]
I don't understand @greenhand-xj

**Reply by greenhand-xj** [reply]
![image](https://github.com/chakra-ui/panda/assets/81698900/6503952e-3e13-46ee-97ce-aeb2a51af755)
A small mistake

**Reply by segunadebayo** [reply]
This is now fixed.

**Reply by greenhand-xj** [reply]
OK haha
